.github/workflows/documentation.yml

```diff
@@ -29,11 +36,24 @@ jobs:
     - name: Install documentation tools
       run: pip install .[docs]
 
+    - name: Install LaTeX packages
+      run: |
+        sudo apt update -y
+        sudo apt install -y \
+          librsvg2-bin \
+          texlive-latex-base \
+          texlive-latex-recommended \
+          texlive-fonts-recommended
+          # texlive-latex-extra
```

Comment:
If this commented code is no longer needed it should be removed.

Reply:
I am not 100% if the docs will require any functionalities from latex-extra. It is moderately heavy install, so I would like to see if latexpdf can handle without this. If not may need to include this. Let me try this in the other workflow...

Reply:
After some testing, it appears that we need texlive-latex-extra and latexmk additional packages (now included).